v0.8.1 release notes (#439)
submariner-io/submariner-website: src/content/community/releases/_index.en.md

weight = 40
+++

## v0.8.1

> * Fix Submariner healtcheck support with Globalnet.

Reviewer: ```suggestion
* Fix Submariner healthcheck support with Globalnet.
```

Author: Done.

> * Fix Submariner healtcheck support with Globalnet.

Reviewer 2: ```suggestion
* Submariner Gateway Health Check is now supported with Globalnet deployments.
```

Author: Done.

> * OVN in kind support deployment support `make clusters using=ovn` (E2E testing & development environment).

Reviewer: ```suggestion
* Support deploying OVN in KIND using `make clusters using=ovn` (E2E testing & development environment).
```

> * OVN in kind support deployment support `make clusters using=ovn` (E2E testing & development environment).

Reviewer 2: ```suggestion
* Added support for kind deployments with OVN Kubernetes CNI (`make clusters using=ovn`).
```

> * Support debugging in Libreswan cabledriver.

Reviewer 2: ```suggestion
* Support debugging in Libreswan cable driver.
```

Author: Done

> * Fix the cabledriver label in the latency metrics.

Reviewer 2: What was fixed exactly?

Author: https://github.com/submariner-io/submariner/pull/1052  We were tagging the metrics with an incorrect label.

> * Support non-tls connections for OVN databases.

Reviewer 2: ```suggestion
* Support non-TLS connections for OVN databases.
```

Author: Done.

> * Services can now be recreated without needing to recreate ServiceExport

Reviewer 2: ```suggestion
* Services can now be recreated without needing to recreate their associated `ServiceExport` objects.
```

Author: Done.

> * Service Discovery no longer requires Submariner connectivity to be available.

Reviewer: IMHO this is a bit confusing. We enhanced Service Discovery to only advertise services in Connected clusters.

I hope the message you want to convey is that "Service Discovery no longer requires Submariner to be available"

Author: Done.

Reviewer 3: Reading this from an outside perspective still confuses me. Do you mean that Lighthouse can be deployed individually?

Author: That it's able to work individually, if you do it via operator, it will work, but subctl support still depends on this https://github.com/submariner-io/submariner-operator/pull/1055  I'll finish it today before moving on to other topics.

> * Service Discovery no longer requires Submariner connectivity to be available.

Reviewer 2: ```suggestion
* Service Discovery no longer depends on Submariner-provided connectivity.
```

Author: Done.

* Improved service discovery verification suite.

> * ServiceImport now includes Port information from the original Service.

Reviewer 2: ```suggestion
* The `ServiceImport` object now includes Port information from the original Service.
```

Author: Done.

weight = 40
+++

## v0.8.1

> * Fix Submariner Gateway Health Check is now supported with Globalnet.

Reviewer 2: ```suggestion
* Submariner Gateway Health Check is now supported with Globalnet deployments.
```

Author: ehhh lol ':D done.

> * Support deploying OVN in KIND using `make clusters using=ovn` (E2E testing & development environment).

Reviewer 2: ```suggestion
* Support deploying OVN in kind using `make clusters using=ovn` for E2E testing & development environment.
```

Author: Done.

> * Support deploying OVN in kind using `make clusters using=ovn` for E2E testing & development environments.

Reviewer 4: ```suggestion
* Support deploying OVN in kind using `make clusters using=ovn` for E2E testing and development environments.
```

> * Support debugging in Libreswan cable driver.

Reviewer 4: ```suggestion
* Support debugging the Libreswan cable driver.
```

* Fix the cable driver label in the Prometheus latency metrics.
* Support non-TLS connections for OVN databases.
* Services can now be recreated without needing to recreate their associated `ServiceExport` objects.

> * Service Discovery no longer depends on Submariner-provided connectivity.
> * Improved service discovery verification suite.

Reviewer 4: ```suggestion
* Service Discovery no longer depends on Submariner-provided connectivity.
* Improved Service Discovery verification suite.
```

Should be consistent.

* The `ServiceImport` object now includes Port information from the original Service.

> * `subctl show` will indicate when the target cluster doesn't have Submariner installed.

Reviewer 2: ```suggestion
* `subctl show` now indicates when the target cluster doesn't have Submariner installed.
```

Author: Done.
